src/features/canvass/components/CanvassPage.tsx

```diff
@@ -51,30 +57,42 @@ const Page: FC<{ assignment: ZetkinAreaAssignment }> = ({ assignment }) => {
             <Box
               alignItems="center"
               display="flex"
-              gap={1}
               justifyContent="space-between"
-              padding={2}
+              py={2}
             >
-              <Box>
-                <Box display="flex" flexDirection="column">
+              <Box display="flex">
+                <ArrowLeftIcon
+                  fontSize="large"
+                  onClick={() => router.push(`/canvass/${assignment.id}/area`)}
+                  sx={{ alignSelf: 'center', cursor: 'pointer', mr: 1 }}
+                />
```

Comment:
I liked this interaction when I tried it, and it made me miss having the same type of button on the list page.